libs/src/services/contracts/ContractClient.js

```diff
@@ -72,13 +72,9 @@ class ContractClient {
       this._isInitializing = false
       this._isInitialized = true
     } catch (e) {
-      // If using ethWeb3Manager or useExternalWeb3 is true, do not do reselect provider logic and fail
-      if (!this.providerSelector) {
-        console.error(`Failed to initialize ${this.contractRegistryKey} contract`, e)
-        return
-      }
-
-      await this.retryInit(selectNewEndpointOnRetry)
```

Comment:
We're not using `selectNewEndpointOnRetry` anymore, should we remove it as an argument to `init`?

Reply:
done!